.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,20 @@
     - Screenshots
```

Comment:
Screenshots are already mentioned here. Adding more text is unlikely to be read by contributors, we want to keep the instructions as minimal and relevant as possible.

Is dark/light theme a common enough topic that it needs to be a permanent part of the PR template?